src/Digdir.Domain.Dialogporten.Infrastructure/HealthChecks/WellKnownHealthChecks.cs

```diff
@@ -0,0 +1,77 @@
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Digdir.Domain.Dialogporten.Infrastructure.HealthChecks;
+
+internal sealed class WellKnownEndpointsHealthCheck : IHealthCheck
+{
+    private readonly IHttpClientFactory _httpClientFactory;
+    private readonly ILogger<WellKnownEndpointsHealthCheck> _logger;
+    private readonly IConfiguration _configuration;
+
+    public WellKnownEndpointsHealthCheck(
+        IHttpClientFactory httpClientFactory,
+        ILogger<WellKnownEndpointsHealthCheck> logger,
+        IConfiguration configuration)
+    {
+        _httpClientFactory = httpClientFactory ?? throw new ArgumentNullException(nameof(httpClientFactory));
+        _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        _configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+    }
+
+    public async Task<HealthCheckResult> CheckHealthAsync(
+        HealthCheckContext context,
+        CancellationToken cancellationToken = default)
+    {
+        var wellKnownEndpoints = _configuration.GetSection("WebApi:Authentication:JwtBearerTokenSchemas")
```

Comment:
Dette er en code smell på at dette ikke ligger på riktig nivå. 
1. Vi burde streve etter å hente ut typed config (InfrastructureSettings / WebApiSettings) med alle fordeler det gir. 
2. Vi henter ut WebApi settings i infrastructure. Infrastructure gjelder for alle presentasjonslag. GraphQL burde ikke ha disse health checkene fordi den hoster ikke well konwn endepunkt.